README.md

```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
+
+
+```python
+print(f"Slide name: {breast_slide.name}")
+print(f"Dimensions at level 0: {breast_slide.dimensions}")
+print(f"Dimensions at level 1: {breast_slide.level_dimensions(level=1)}")
+print(f"Dimensions at level 2: {breast_slide.level_dimensions(level=2)}")
+```
+
+    Slide name: 9c960533-2e58-4e54-97b2-8454dfb4b8c8
+    Dimensions at level 0: (96972, 30681)
+    Dimensions at level 1: (24243, 7670)
+    Dimensions at level 2: (6060, 1917)
+
+
+
+```python
+print(f"Slide name: {heart_slide.name}")
+print(f"Dimensions at level 0: {heart_slide.dimensions}")
+print(f"Dimensions at level 1: {heart_slide.level_dimensions(level=1)}")
+print(f"Dimensions at level 2: {heart_slide.level_dimensions(level=2)}")
+```
+
+    Slide name: JP2K-33003-2
+    Dimensions at level 0: (32671, 47076)
+    Dimensions at level 1: (8167, 11769)
+    Dimensions at level 2: (2041, 2942)
+
+
+
+```python
+breast_slide.save_thumbnail()
+print(f"Thumbnails saved at: {breast_slide.thumbnail_path}") 
+heart_slide.save_thumbnail()
+
+print(f"Thumbnails saved at: {heart_slide.thumbnail_path}") 
+```
+
+    Thumbnails saved at: processed/thumbnails/9c960533-2e58-4e54-97b2-8454dfb4b8c8.png
+    Thumbnails saved at: processed/thumbnails/JP2K-33003-2.png
+
+
+
+```python
+breast_slide.show() 
+heart_slide.show()
+```
+
+![caption](images/thumbnails.png)
+
+### Tiles extraction
+
+Now that your slide object is defined, you can automatically extract the tiles. ```RandomTiler``` method crops random tiles from the slide.
+You need to specify the size you want your tiles, the number of tiles to crop, and the level of magnification. If ```check_tissue``` is True, the exracted tiles are taken by default from the **biggest tissue region detected** in the slide, and the tiles are saved only if they have at least 80% of tissue inside.
+
+
+```python
+from histolab.tiler import RandomTiler
+
+random_tiles_extractor = RandomTiler(tile_size=(512,512), n_tiles=6, level=2, seed=42, check_tissue=True, prefix='processed/breast_slide/')
+
+random_tiles_extractor.extract(breast_slide)
+```
+
+    	 Tile 0 saved: processed/breast_slide/tile_0_level2_70536-7186-78729-15380.png
+    	 Tile 1 saved: processed/breast_slide/tile_1_level2_74393-3441-82586-11635.png
+    	 Tile 2 saved: processed/breast_slide/tile_2_level2_82218-6225-90411-14420.png
+    	 Tile 3 saved: processed/breast_slide/tile_3_level2_84026-8146-92219-16340.png
+    	 Tile 4 saved: processed/breast_slide/tile_4_level2_78969-3953-87162-12147.png
+    	 Tile 5 saved: processed/breast_slide/tile_5_level2_78649-3569-86842-11763.png
+    	 Tile 6 saved: processed/breast_slide/tile_6_level2_81994-6753-90187-14948.png
+    6 Random Tiles have been saved.
+
+
+![caption](images/breast.png)
+
+
+```python
+random_tiles_extractor = RandomTiler(tile_size=(512,512), n_tiles=6, level=0, seed=42, check_tissue=True, prefix='processed/heart_slide/')
```

Comment:
line too long

Reply:
How can I do?

Reply:
write it in a random py script and blackify it!